bitnami/thanos/Chart.yaml

```
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
   - https://thanos.io
-version: 8.2.5
+version: 8.2.6
```

For changes in the README it's not needed to bump the chart version
```suggestion
version: 8.2.5
```